Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -49,6 +49,7 @@
 #endif
 
 internal import _FoundationCShims
+import Builtin
```

Comment:
Should this be `internal import Builtin`?

Reply:
The Builtin module is public by virtue of being inlined all over the standard library; in this file we're calling it from opaque code, so there's nothing being reexported at all.